packages/react/src/select/trigger/SelectTrigger.tsx

```diff
@@ -225,12 +224,15 @@ export const SelectTrigger = React.forwardRef(function SelectTrigger(
     validation.getValidationProps,
     elementProps,
     getButtonProps,
+    {
+      // Set the role last to ensure nested useButton does not overwrite it
+      // e.g. <Toolbar.Button render={<Select.Trigger />} />
+      role: 'combobox',
+      // combobox box doesn't need the type="button" attribute
+      type: undefined,
```

Comment:
Typo here. So browsers read `role="combobox"` and don't make it a submit button implicitly?